x-pack/plugins/maps/public/actions/data_request_actions.ts

```diff
@@ -67,6 +68,7 @@ export type DataRequestContext = {
   dataFilters: DataFilters;
   forceRefreshDueToDrawing: boolean; // Boolean signaling data request triggered by a user updating layer features via drawing tools. When true, layer will re-load regardless of "source.applyForceRefresh" flag.
   isForceRefresh: boolean; // Boolean signaling data request triggered by auto-refresh timer or user clicking refresh button. When true, layer will re-load only when "source.applyForceRefresh" flag is set to true.
+  isEditingFeatures?: boolean;
```

Comment:
Can you make `isEditingFeatures: boolean;`. Value should always be provided.

Can you add the comment 
```
// Boolean signaling that feature editor menu is open for a layer. When true, layer will ignore all global and layer filtering so drawn features are displayed and not filtered out.
```